CMakeLists.txt

```diff
@@ -75,37 +69,87 @@ target_include_directories(libLinescaleGUI PUBLIC
   $<BUILD_INTERFACE:${AUTOGEN_INCLUDE_DIR}>
 )
 
+# Set up dependencies.
+#
+# We add most dependencies only to `libLinescaleGUI` but since they're public
+# all dependent targets will get the too.
+set(QT_DEPENDENCIES
+    Qt::Core
+    Qt::Widgets
+    Qt::SerialPort
+    Qt::Bluetooth
+    Qt::PrintSupport)
 target_link_libraries(linescaleGUI PRIVATE libLinescaleGUI)
 target_compile_options(linescaleGUI PRIVATE ${warning_compile_options})
 target_link_libraries(libLinescaleGUI PUBLIC ${QT_DEPENDENCIES} qcustomplot)
 target_compile_options(libLinescaleGUI PRIVATE ${warning_compile_options})
 target_compile_definitions(libLinescaleGUI PUBLIC $<IF:$<CONFIG:Debug>,,QT_NO_DEBUG_OUTPUT>)
 target_compile_definitions(linescaleGUI PUBLIC $<IF:$<CONFIG:Debug>,,QT_NO_DEBUG_OUTPUT>)
 
-set_target_properties(linescaleGUI PROPERTIES
-    MACOSX_BUNDLE_GUI_IDENTIFIER linegrip.com
-    MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION}
-    MACOSX_BUNDLE_SHORT_VERSION_STRING ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
-    MACOSX_BUNDLE TRUE
-)
+# Disable the console window for release builds on Windows.
 if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
     set_target_properties(linescaleGUI PROPERTIES
         WIN32_EXECUTABLE TRUE)
+    set(CMAKE_WIN32_EXECUTABLE ON)
 endif()
 
+# Deploy Qt libraries.
 include(tools/cmake/windeployqt.cmake)
 if(WIN32)
     windeployqt(linescaleGUI)
 endif()
 
-# # Copy LICENSE.md into build directory
-add_custom_command(
-    TARGET linescaleGUI POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy
-    ${CMAKE_SOURCE_DIR}/LICENSE.md
-    ${CMAKE_CURRENT_BINARY_DIR}/LICENSE.md
-    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE.md)
-
```

Comment:
Note for later: Replaced by https://github.com/linescaleGUI/linescaleGUI/blob/0c5bf4866de18102304ead8b76a62c9c36c93016/tools/cmake/packaging.cmake#L59-L59 within the same PR.